### ISSUE TITLE
Container are started in wrong order

### ISSUE THREAD

**Issue by Luzifer** [Luzifer/dockermanager#2]
When containers are linked the containers are probably started in wrong order: There is no dependency graph inside dockermanager so it does not know which container needs which container to be started.

```
Jul 07 07:35:32 fwwedel01 dockermanager[851]: time="2017-07-07T07:35:32+02:00" level=error msg="Unable to create container 'jenkins': API error (500): {\"message\":\"Could not get container for crowd\"}\n"
```

Additionally things break when linked containers are restarted so when restarting a linked container all containers linked to that one needs to be restarted too.

**Comment by Luzifer** [comment]
Starting in wrong order should be fixed in v1.2.0-rc1 - pending testing

Breaking links through container restarts not yet addressed

**Comment by Luzifer** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Dockermanager v1.2.0](https://trello.com/c/HHKx1XY6/118-dockermanager-v120)

**Comment by Luzifer** [comment]
Resolved in v1.2.1
